common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java

```diff
@@ -130,18 +132,28 @@ public final void putString(
         if (null == mStorageHelper) {
             editor.putString(key, value);
         } else {
-            editor.putString(key, encrypt(value));
+            final String encryptedValue = encrypt(value);
```

Comment:
In an upcoming PR, `null` may be used to signal an Exception case. For now, preserving existing behavior